### PR TITLE
Add a URL to 'You Might Not Need Redux'

### PR DESCRIPTION
**Pull request by mesaugat**
[You Might Not Need Redux](https://medium.com/@dan_abramov/you-might-not-need-redux-be46360cf367#.eqv7ilqo9)

This article sufficiently states why you might not need Redux from the creator itself. Adding this to the documentation will help those people who are not able to decide whether they need Redux or not.

**Not sure if it is the correct place for the link, please suggest.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,11 @@ It is tiny (2kB, including dependencies).
 >[“It's cool that you are inventing a better Flux by not doing Flux at all.”](https://twitter.com/andrestaltz/status/616271392930201604)  
 >André Staltz, creator of Cycle
 
+### Before Proceeding Further
+
+>**Also read why you might not be needing Redux:
+>[“You Might Not Need Redux”](https://medium.com/@dan_abramov/you-might-not-need-redux-be46360cf367#.eqv7ilqo9)**
```
Comment (gaearon) [inline README.md]: Let’s remove the annoying hash part of URL

File: README.md [hunk 2]
```diff
@@ -29,6 +29,11 @@ It is tiny (2kB, including dependencies).
 >[“It's cool that you are inventing a better Flux by not doing Flux at all.”](https://twitter.com/andrestaltz/status/616271392930201604)  
 >André Staltz, creator of Cycle
 
+### Before Proceeding Further
+
+>**Also read why you might not be needing Redux:
```
Comment (gaearon) [inline README.md]: If you intend to break the line here, add two spaces after the line end. (Yes, Markdown is weird!)

File: README.md (hunk 1 above)
Comment (mesaugat) [inline README.md]: Sure

File: README.md (hunk 2 above)
Comment (mesaugat) [inline README.md]: Coming up

**Comment by mesaugat** [comment]
It's done. Looks like [force-push] caused some trouble. (https://github.com/reactjs/redux/pull/1995/files/d4948a4d7550fce27dd973f67ca5cede3f141d03)

The files have been changed.

**Comment by timdorr** [comment]
Thanks!

**Pull request merged**